TictacToe/TictacToe/Program.cs

```diff
@@ -23,21 +21,37 @@ static void Main(string[] args)
             int turns = 0;
             int maxTurns = 9;
             bool gameComplete = false;
+            bool isEmpty = true;
```

Comment:
is empty statusu būtu vērtīgāk uzturēt spēles laukuma klasē.

spēles laukuma klase ir atbildīga par spēles laukuma statusu, attiecīgi, manuprāt, viņa varētu zināt vai laukums ir tukšs vai nav.